source/default/_patterns/00-protons/index.js

```diff
@@ -1,9 +1,32 @@
 /**
- * Base css generation and global js logic
+ * Base css generation and global js logic.
  */
 
 import './_base.scss';
 
+import { mediaBreakpoint } from 'protons/utilities';
+import scssVariables from '../../_data/scssVariables.json';
+
+import enquire from '../../../node_modules/enquire.js';
+
+export function enable() {
+  // Example usage of the Enquire.js module JS breakpoints.
+  const { GLOBAL_BREAKPOINTS } = scssVariables;
```

Comment:
Let's take the approach of reading the CSS variables: https://stackoverflow.com/a/41725782

Reply:
@illepic since the CSS variables were all named in ways that make it impossible to actually loop through them to assign to JS, I've also taken the liberty of making a new CSS variable that holds all of the breakpoint names. This is located in `printing/_elements.css`, but I'm not married to that.